packages/aws-cdk-lib/core/README.md

```diff
@@ -1650,4 +1650,70 @@ warning by the `id`.
 Annotations.of(this).acknowledgeWarning('IAM:Group:MaxPoliciesExceeded', 'Account has quota increased to 20');
 ```
 
+## RemovalPolicys
+
+The `RemovalPolicys` class provides a convenient way to manage removal policies for AWS CDK resources within a construct scope. It allows you to apply removal policies to multiple resources at once, with options to include or exclude specific resource types.
+
+### Usage
+
+```typescript
+import { RemovalPolicys } from 'aws-cdk-lib';
+
+// Apply DESTROY policy to all resources in a scope
+RemovalPolicys.of(scope).destroy();
+
+// Apply RETAIN policy only to specific resource types
+RemovalPolicies.of(parent).retain({
+  applyToResourceTypes: [
+    'AWS::DynamoDB::Table',
+    bucket.cfnResourceType, // 'AWS::S3::Bucket'
+  ],
+});
+
+
+
+// Apply SNAPSHOT policy excluding specific resource types
+RemovalPolicys.of(scope).snapshot({
+  excludeResourceTypes: ['AWS::Test::Resource'],
+});
+
+// Apply RETAIN_ON_UPDATE_OR_DELETE policy
+RemovalPolicys.of(scope).retainOnUpdateOrDelete();
+```
+
+### RemovalPolicys.of(scope)
+
+Creates a new instance of RemovalPolicys for the given scope.
+
+#### Methods
+
+- `apply(policy: RemovalPolicy, props?: RemovalPolicyProps)`: Apply a custom removal policy
+- `destroy(props?: RemovalPolicyProps)`: Apply DESTROY removal policy
+- `retain(props?: RemovalPolicyProps)`: Apply RETAIN removal policy
+- `snapshot(props?: RemovalPolicyProps)`: Apply SNAPSHOT removal policy
+- `retainOnUpdateOrDelete(props?: RemovalPolicyProps)`: Apply RETAIN_ON_UPDATE_OR_DELETE removal policy
+
+#### `RemovalPolicyProps` Interface
+
+Additional configuration options for applying removal policies.
+
+- **`applyToResourceTypes?`**: _(optional)_  
+  Array of CloudFormation resource types (e.g., `'AWS::S3::Bucket'`) to which the removal policy should be applied.  
+  Defaults to applying to all resources.
+
+- **`excludeResourceTypes?`**: _(optional)_  
+  Array of CloudFormation resource types to exclude from applying the removal policy.  
+  Defaults to no exclusions.
+
+- **`overwrite?`**: _(optional)_  
+  If `true`, the removal policy will overwrite any existing policy already set on the resource. Defaults to `false`.
```

Comment:
Wouldn't it be difficult to maintain if all properties were listed on the document?

Reply:
clean up this doc.
https://github.com/aws/aws-cdk/pull/32283/commits/da6b4c59406bca4d068fb60747b1a7616bcc9081